core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -1054,4 +1058,21 @@ public static boolean hidePassengerMutations() {
 	String hidePassenger = properties.getProperty(HIDE_PASSENGER_MUTATIONS, "false");
 	return Boolean.parseBoolean(hidePassenger);
     }
+    
+    public static String getQuerySetsOfGenes() {
+	String result = new String();
+	try {
+	    BufferedReader br = new BufferedReader(new FileReader(ResourceUtils.getFile(properties.getProperty(SETSOFGENES_LOCATION, "classpath:/setsofgenes.json"))));
+	    for (String line = br.readLine();line != null;line = br.readLine()) {
+		line = line.trim();
+		result = result + line;
+	    }
+	    br.close();
+	} catch (FileNotFoundException e) {
```

Comment:
@oplantalech is good to have a comment here explaining why we catch and re-throw as RuntimeException: because of the way the getQuerySetsOfGenes method is used in .jsp it needs to be static and plain (no "throws" in signature).